harness/determined/experimental/_native.py

```diff
@@ -287,30 +277,33 @@ def create(
     Arguments:
         trial_def:
             A class definition implementing the ``det.Trial`` interface.
+
         config:
             A dictionary representing the experiment configuration to be
             associated with the experiment.
-        mode:
-            The :py:class:`determined.experimental.Mode` used when creating
-            an experiment
 
-            1. ``Mode.CLUSTER`` (default): Submit the experiment to a remote
-            Determined cluster.
+        local:
+            A boolean indicating if training should happen locally. When
+            ``False``, the experiment will be submitted to the Determined
+            cluster. Defaults to ``False``.
 
-            2. ``Mode.LOCAL``: Test the experiment in the calling
-            Python process for local development / debugging purposes.
-            Run through a minimal loop of training, validation, and checkpointing steps.
+        test:
+            A boolean indicating if the experiment should be shortened to a
```

Comment:
1. "should" -> "will"
2. 
> ``test=True`` is useful quick iterating during model porting or debugging.

Can you expand this sentence?

Reply:
```
            ``test=True`` is useful quick iterating during model porting or
            debugging because many common failures will be exposed more
            quickly. Defaults to ``False``.
```
How is that?

Reply:
perfectly good